### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by joe247**
Please review the updated ReadMe

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -37,7 +37,23 @@ Features
...
   • Predicts Unicode symbols and emoji as well.
 
-━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+## Requirements
+  • Some packages are requied if you are building from source:
+    ┏━━━━━━━━━━━━━━━━━━━┯━━━━━━━━━┓
+    ┃     Package       │ Version ┃
+    ┠───────────────────┼─────────┨
+    ┃ ibus              │ 1.5.3+  ┃
+    ┃ libm17n-0         │ 1.7.0+  ┃
```
Comment (mike-fabian) [inline README]: The package  names  depend  on  the distribution. 
See: http://mike-fabian.github.io/ibus-typing-booster/development.html

File: README [hunk 2]
```diff
@@ -159,18 +166,18 @@ The following table explains the defaults:
 ┃                   │ preëdit text.                                           ┃
 ┠───────────────────┼─────────────────────────────────────────────────────────┨
 ┃ Delete            │ Remove the typed key to the right of the cursor in the  ┃
-┃                   │ preëdit text.                                           ┃
+┃ <kbd>⌫</kbd>      │ preëdit text.                                           ┃
 ┠───────────────────┼─────────────────────────────────────────────────────────┨
 ┃ Control+Delete    │ Remove everything to the right of the cursor in the     ┃
-┃                   │ preëdit text.                                           ┃
+┃ <kbd>⌃⌫</kbd>     │ preëdit text.                                           ┃
```
Comment (mike-fabian) [inline README]: This doesn't  look like Control+Delete. 

By the way, I  autogenerate this section explaining the key bindings  by viewing http://mike-fabian.github.io/ibus-typing-booster/documentation.html#key-bindings with the  w3m text browser to  keep the README and the online documentation in sync.

File: README (hunk 1 above)
Comment (mike-fabian) [reply README]: Maybe  just point to the online documentation instead of putting this into the REAME?

File: README (hunk 2 above)
Comment (joe247) [reply README]: Oh, I see... so what do I do then, should I leave every thing as it is and close the pull request or make some changes?

(*I'm not much familiar with git*)

Comment (mike-fabian) [reply README]: No problem! I can work a bit on this. But not next week, the week after that.
You want to to convert README into README.md and add some markup  to make it look better on github. I think I can use some of your changes, I’ll put all the useful ones into one commit with your name the week after next week.

Comment (joe247) [reply README]: Alright!

Comment (joe247) [reply README]: Alright!

**Pull request closed without merging**